.golangci.yaml

```diff
@@ -35,6 +35,7 @@ issues:
   - indent-error-flow # if block ends with a return statement, so drop this else and outdent its block
   - "exported: (type|func) name will be used as .* by other packages, and that stutters;"
   - unused-parameter # parameter '.*' seems to be unused, consider removing or renaming it as _
+  - 'duplicated-imports: Package "embed" already imported' # Allow duplicated import for "embed" package
```

Comment:
Why do we allow this? 👀

Reply:
Hmm, not sure. It looks like we can remove the empty import in `pkg/provider-local/charts/charts.go`. Adapted.

Reply:
I was thinking if we had a variable like,
```
	//go:embed images.yaml
	imagesYAML  string
```
as well, then, the empty import is required. But maybe not.

Reply:
OK, but how is this related to allowing duplicate embed import? 😄